renovate.json

```diff
@@ -11,13 +11,13 @@
         "*"
       ],
       "enabled": false
-    },
+    }, 
     {
       "matchPackagePatterns": [
-        "^com.google.api:gapic-generator-java",
```

Comment:
Why do we need to remove `com.google.api:gapic-generator-java` from this group?

Reply:
Because the title of the grouped dependency update PR would not include versions. e.g. https://github.com/googleapis/google-cloud-java/pull/11271. In additional, even though they are from the same repo, `gapic-generator-java` and `sdk-platform-java-config` have different versions, so I think we should keep them separate.